packages/appstore/src/components/wallet-account/wallet-account.scss

```diff
@@ -0,0 +1,108 @@
+.wallet-account {
+    position: relative;
+    background: #fff8f9;
+
+    &__background {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        height: 15.8rem;
+    }
+
+    &__header {
+        height: 100%;
+        position: relative;
+        border-radius: 1.6rem;
+        overflow: hidden;
+        padding: 2.4rem;
+        display: flex;
+        align-items: center;
+    }
+
+    &__apps-wrapper {
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__apps-title {
+        display: flex;
+        align-items: center;
+        margin-top: 2rem;
+        margin-bottom: -1rem;
+
+        &--link {
+            margin-left: 1.5rem;
+        }
+
+        &.no-mt {
+            margin-top: 0 !important;
+        }
+    }
+
+    &__apps {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 3rem;
+    }
+
+    &__app-launcher-button {
+        background-color: #ff444f29 !important;
+    }
+
+    &__details {
+        display: flex;
+        flex: 1;
+    }
+    &__logo {
+        margin-right: 2.4rem;
+    }
+    &__info {
+        display: flex;
+        flex-direction: column;
+        flex: 1;
+    }
+    &__actions {
+        display: flex;
+        flex: 1;
+        margin-top: 1rem;
+        gap: 8px;
+    }
+    &__balance {
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__content {
+        padding: 2.1rem;
+        overflow: hidden;
+    }
+
+    &__deriv-apps {
+        background-color: #fff0f1;
+        border-radius: 16px;
+        padding: 0 24px 24px;
+        margin-top: 24px;
+    }
+
+    &__app-launcher--full-width {
+        margin: 1.6rem 0;
+        min-height: 9.6rem;
+        display: flex;
+        gap: 0.8rem;
+        align-items: center;
+        border-bottom: 1px solid #fff;
+    }
+
+    &__app-card-wrapper {
+        display: flex;
+        justify-content: center;
+        padding: 3rem;
+        flex-direction: column;
+        align-items: center;
+        gap: 3rem;
+    }
+}
+
+.link-text {
+    color: #ff444f !important;
```

Comment:
This color is available as `--brand-red-coral` in shared, can we use it from there?

Reply:
done :heavy_check_mark: